tests/main/snap-confine/task.yaml

```diff
@@ -0,0 +1,15 @@
+summary: Port of the snap-confine integration tests
```

Comment:
This test (as is) is not needed, there is a better set of tests in `tests/main/snap-seccomp` which already tests a bunch of the above cases, i.e. invalid syntax is not compiled, valid syntax works and snap-confine can work with the generated bpf.